src/core/InputSlider/style.ts

```diff
@@ -0,0 +1,198 @@
+import styled from "@emotion/styled";
+import { Slider, SliderProps } from "@material-ui/core";
+import {
+  CommonThemeProps,
+  fontBodyXxs,
+  getColors,
+  getCorners,
+} from "../styles";
+
+export interface InputSliderExtraProps extends SliderProps, CommonThemeProps {}
+
+const inputSlider = (props: InputSliderExtraProps) => {
+  const corners = getCorners(props);
+  const colors = getColors(props);
+
+  return `
+  position: relative;
+
+  .MuiSlider-rail,
+  .MuiSlider-track {
+    border-radius: 2px;
+  }
+
+  .MuiSlider-rail {
+    background-color: ${colors?.gray[200]};
+  }
+  
+  .MuiSlider-track {
+    background-color: ${colors?.primary[400]};
+  }
+
+  .MuiSlider-thumb {
+    height: 14px;
+    width: 14px;
+    background-color: ${colors?.primary[400]};
+
+    &.Mui-focusVisible, &:hover, &:focus, &:active {
+      box-shadow: none;
+    }
+  }
+
+  .MuiSlider-thumb::after {
+    background-color: white !important;
+    height: 6px !important;
+    width: 6px !important;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  .MuiSlider-valueLabel {
+    padding: 2px 4px;
+    color: black;
+    background-color: ${colors?.primary[200]};
+    border-radius: ${corners?.m}px;
+    left: unset; 
+    top: -15px;
+
+    & * {
+      background: transparent;
+      color: black;
+      transform: none;
+      width: unset;
+      height: unset;
+      font-size: 12px;
+      font-weight: 400;
+      letter-spacing: 0.3px;
+      line-height: 18px;
+    }
+  }
+
+  .MuiSlider-mark {
+    background-color: ${colors?.gray[400]};
+    opacity: 1;
+  }
+
+  .MuiSlider-mark.MuiSlider-markActive {
+    background-color: white;
+  }
+
+  .MuiSlider-markLabel {
+    font-size: 12px;
+    font-weight: 400;
+    letter-spacing: 0.3px;
+    line-height: 18px;
+    color: ${colors?.gray[500]}
+  }
+
+  .MuiSlider-markLabelActive {
+    color: black;
+  }
+  `;
+};
+
+const horizontal = () => {
+  return `
+    .MuiSlider-rail,
+    .MuiSlider-track {
+      height: 4px;
+    }
+
+    .MuiSlider-rail {
+      padding-right: 3px;
+    }
+    
+    .MuiSlider-track {
+      padding-right: 3px;
+    }
+
+    .MuiSlider-mark {
+      margin-top: 1px; 
+      margin-left: 1px;
+    }
+
+    .MuiSlider-mark ~ .MuiSlider-mark {
+      margin-left: 0;
+    }
```

Comment:
Not 100% sure, but the following seems to work?

My thought is that all `.MuiSlider-mark` elements can be `margin-left: 0`, including the first one. But not sure if there's a specific reason the first one should be `margin-left: 1px`

Thank you!

```suggestion
    .MuiSlider-mark {
      margin-top: 1px; 
      margin-left: 0;
    }
```

Reply:
We need a margin-left of 1px for the first mark element only. Otherwise, it would stick to the rail element.
<img width="785" alt="Screen Shot 2022-05-31 at 4 18 25 PM" src="https://user-images.githubusercontent.com/927990/171277892-fbf93bce-ba98-4b38-907d-4a4398f0bbe4.png">

Reply:
Ahh got it! Thanks for the screenshot 🙏 

How about:

```css
.MuiSlider-mark {
  margin-top: 1px; 
  margin-left: 0;
}

// the first mark needs 1px margin-left to not touch the rail element
.MuiSlider-mark[data-index="0"] {
  margin-left: 1px;
}
```

I'm hoping this improves the readability and intention of the styling. Thank you!

Reply:
So brilliant 🤩 
Thanks for sharing your awesome thoughts 🙏🏻

Reply:
Awesome!! So glad that looks good to you too 🙆‍♂️ Thanks for your great attention to details 🔥 🦅 👁️